addons/forum-post-countdown/userscript.js

```diff
@@ -0,0 +1,37 @@
+export default async function ({ addon, msg }) {
+  const submitButton = document.querySelector("#djangobbwrap .form-submit [type=submit]");
+
+  submitButton.addEventListener("click", () => {
+    if (!localStorage.getItem("sa-forum-post-countdown")) {
+      localStorage.setItem("sa-forum-post-countdown", Date.now());
+    }
+  });
+
+  const countdown = localStorage.getItem("sa-forum-post-countdown");
+  if (!countdown) {
+    return;
+  }
+
```

Comment:
Ideally, at this point of the code we would check if the countdown is already overdue (for example, it might have been 24 hours), delete the storage item and return in that case.

Reply:
I think that would avoid the countdown from showing for 1 second then disappearing. Haven't actually tried to test or emulate this case, though

Reply:
With the countdown now being in the post button, is this actually that important? Worst case, there's a bit too much padding on the submit button for a second.